### ISSUE TITLE
workouts routes just shows the number of exercises and after clicking on edit(?) it shows exercises, it's expected to show all exercise in the main page

### ISSUE THREAD

**Issue by pouyanze** [Lo-Agency-Internship/samira-shabnam-khashayar-workout-tracker#35]

**Comment by Khashiiiiiii** [comment]
@pouyanze 
Hello there, I didn't quite understand what did you mean by this issue. Can you please explain it more?
Thanks

**Comment by pouyanze** [comment]
yes, for the user to be able to view their exercises, the edit button should be clicked

**Comment by Khashiiiiiii** [comment]
> yes, for the user to be able to view their exercises, the edit button should be clicked

Yes, that is the point. When the user logs in, the ```workout``` page appears, on this page they can see a card for themselves and bunch of other cards for every user that have been signed up and created some exercises. With the edit button he will be redirected to ```modify``` page where he can delete, add or edit and exercise. The whole structure of our project was designed to be this way.
